schematic/lepton-schematic.scm

```diff
@@ -1,11 +1,12 @@
 #!/usr/bin/env sh
 export GUILE_LOAD_COMPILED_PATH="@ccachedir@:${GUILE_LOAD_COMPILED_PATH}"
+export GUILE_LOAD_PATH="@LEPTON_SCHEME_MODULE_DIRECTORY@:${GUILE_LOAD_PATH}"
```

Comment:
Now we have 3 (three) `$PREFIX/share/lepton-eda/scheme`
entries in the `%load-path`. :-)

Reply:
Well, yes. But at the time it is needed, before initialization of `liblepton` extension there is only one defined in that code.  I could not managed to make the module work any other way.  It doesn't wish to be loaded from `primitive-eval` for some reason.